---
title: "Analysis Report 2: Gender-based Prognostic Analysis of Known Gene Drivers in Patients with Lung adenocarcinoma"
author: "Alfredo Hans Locht"
date: "November 10, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

## Background

Lung cancer has been identified worldwide as the most prevalent type of cancer with the highest mortality rates in developed countries [@Sakurai2010; @li2015rna; @seo2012transcriptional]. Tobacco cigarette smoking has been identified as the main contributer, along with a host of other genetic and environmental factors, to cause Lung adenocarcinoma, the most common type of non-small cell lung cancer (NSCLC). Genetic research in this field has mainly focused on finding the gene-expression differences between non-smokers and smokers with NSCLC and even between smokers who have and have not contracted any form of NSCLC [@Cheng2012; @li2015rna; @seo2012transcriptional]. These studies have found significant gene expression profile differences between smokers and nonsmokers with NSCLC; however, these studies have not addressed gender-based expression differences.

Although there has been an over 500% increase in lung cancer rates among women since the 1980's, it is becoming increasingly evident that women contain gender-based genetic differences that predicate a favorable prognosis and higher survival rate than men when normalizing for other factors [@Chatkin2004; @Sakurai2010]. These gender-based differences have only been observed on clinicopathological basis with not much emphasis on the genetic data that may be available for such an analysis. Although common grounds have not been genetically established for a gender-based analysis of NSCLC, the disease has been able to be characterized genetically, by mutational analysis.

Given how 60 of the known carcinogens in tobacco smoke can actively mutate cells to become cancerous, there have been several somatic point mutations as well as aberrant fusion gene products identified specifically in NSCLC [[@Al-Saad2017; @Cheng2012; @seo2012transcriptional; @Yokota2000]. These genes can serve as a basis for a potential gender-based NSCLC analysis because they are not gender-discriminate, and because the resulting biology from their mutations is relatively well understood.

## Objective

In this study, to emphasize that there are key distinctions between male and female gene expression profiles, genes that have been identified through somatic point mutations and contributing to malignency in NSCLC will be analyzed per the Li et al dataset and metadata [@Al-Saad2017; @li2015rna]. Specifically, the genes that will be analyzed are Epithelial growth factor receptor (EFGR), Kirsten rat sarcoma virus (KRAS), mesenchymal-epithelial transition factor (MET) as well as insulin-like growth factor 1 receptor (IGFR1), all of which are potential therapeutic targets against advancing malignancy. It is also important to note that EFGR1, IGFR1 and MET (more commonly known as c-Met or HGFR) not only play critical roles in activating pathways such as RAS-MAPK pathway, but are also characterized as important in patient-developed treatment resistance [@Al-Saad2017]. Furthermore, Al-Saad et al. describes that high expression of MET and EFGR1 (either through high gene copy or abberant transcription amplification) are indicative biomarkers of a negative prognosis in NSCLC [@Al-Saad2017]. The expression of these genes will be divided by gender, age at diagnosis, and cancer stage to elucidate how women have gene expression differences that contribute to a better prognosis in NSCLC.


# Methods

## Sample origin and sequencing

The dataset generated by Seo et al. for their journal article, The transcriptional landscape and mutational profile of lung adenocarcinoma, was obtained by collecting 200 specimens from 87 patients inflicted with lung adenocarcinoma who had undergone major lung resection surgeries at Saint Mary's Hospital in Seoul, South Korea. The patients provided their smoking history which included smokers (47), non-smokers (36), and some who did not not disclose their smoking history (4). All of the samples originated from people of Korean ethnicity, and out of the 200 specimen, 87 were identified as cancerous stemming from driver mutations while 77 samples were identified to be normal tissue as a basis for comparison of gene expression profiles against the cancerous samples. Together, these 164 were RNA-sequenced using an Illumina HiSeq 2000 sequencer yielding ~ 14 billion paired-end, 101-bp long reads [@seo2012transcriptional]. Additionally, genomic DNA was extracted from normal lung tissue for exome sequencing. Together, the RNA and exome sequencing reads were aligned to the NCBI human reference genome (build 37.1) using GSNAP with a 5% allowance for mismatches. Image analysis and basecalling was performed using the Illumina pipeline on the default settings (v1.8).

Li et al. obtained 68 lung adenocarcinoma patient transcriptome sequencing data from the aforementioned Seo et al. study database. Each patient had their tumor and normal tissue sequenced comparitively, and the total amount and length of paired-end sequences mimicked the Seo et al. study because it was the same data. However, they normalized the data to be an equal amount of non-smokers (34) vs smokers (34) by exluding the data of patients over the age of 75 to ensure equal comparing grounds [@li2015rna].

## Computational

These are the methods that were used to process the sequencing data. Should probably be at least a half of a page. At a very minimum should include citations for biomartr, trimmomatic, and sailfish. Note that these three methods references don't count towards the five references you need to cite in the introduction.

# Results

Data was subsetted from Li et al. to create all the figures in the Results section. Importantly to note is that each gene that was analyzed (EGFR, KRAS, MET and IGF1R) has exactly the same number of samples (76) that were divided into 31 females and 45 males across all genes. This indicates that the results are comparable because essentially the same sub-sample set was used for each figure created. The data that was subsetted for the genes of interest however, failed to include any females in stage 2B, so there is no representation of females for that cancer stage for any gene of interest.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```
**Figure 1 Description**: Figure 1 shows the top 15 most highly expressed genes per gender.

```{r EFGR Expression at Different Canceer Stages}
# Seo et al. discusses that 79% of their patients' NSCLC is driven by point mutations in EFGR.
EGFR <- final_table %>%
  filter(genename == "EGFR") %>%
  group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

EGFR %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))

```
**Figure 2 Description**: Figure 2 shows EFGR expression according to gender through different cancer stages. EFGR expression is overall higher in males of any age at any cancer stage. The borders dividing the columns horizontally represent each gender at a specific cancer stage by age range, so it is evident that EFGR expression does not follow an age-trend in males as the high expression in stage 3A shows most of the males to be in their 80s while the high expression in stage 3B indicates that the population of males with high EFGR expression is closer to their 40s. Female expression of EFGR across all cancer stages is mostly inclusive to older women in their 70s and 80s.  

```{r KRAS Expression at Different Canceer Stages}
KRAS <- final_table %>%
  filter(genename == "KRAS") %>%
  group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

KRAS %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))

```
**Figure 3 Description**: Figure 3 shows KRAS expression according to gender through different cancer stages. KRAS expression across cancer stages is much higher than EFGR for both sexes. In general, both sexes have a much higher expression of KRAS in stages 1A-2B than they do in stages 3A-4 (not including NA cancer stage). There is also more vairance in the age ranges as it appears that all the age ranges experience KRAS expression in stages 1A and 1B. Females; however, show less expression of KRAS overall.

```{r MET Expression at Different Canceer Stages}
MET <- final_table %>%
  filter(genename == "MET") %>%
group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = 
              mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) 

MET %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge",
             aes(color = age_at_diagnosis)) 
  
```
**Figure 4 Description**: MET expression is shown in figure 4, and it can be seen that stage 1A for males and stage 1B for females resembles the KRAS expression in Figure 3 in terms of overall reads and age division reprentation. Overall, MET expression is higher in males in the later stages of cancer (except for 1A), but there is no trend in age factors. Conversely, MET expression is lower in females in later stages of cancer, but there is no age trend either. Female expression of MET is significantly lower overall whenc compared to male expression of MET across all cancer stages.

```{r IGF1R Expression at Different Canceer Stages}
IGF1R <- final_table %>%
  filter(genename == "IGF1R") %>%
  group_by(gender, 
           age_at_diagnosis, 
           cancer_stage) %>%
  summarize(mean_count = 
              mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

IGF1R %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))
```
**Figure 5 Description**: Expression of IGF1R gives a much more detailed look at expression in females vs males than the rest of the genes because expression is much higher overall. Importantly, we see that male expression is much higher overall than any particular female expression at a specific stage. For males, the columns are more segmented across the stages which indicates that all age ranges are represented in male IGF1R expression. In comparison, females are predominantly more dominated by older age ranges.

# Discussion

Limitations: We have the patient data, but do not know if they have had any kind of previous operation to extract or get rid of the cancer in the metadata or what kind of treatment (if any) they have undergone. --> nevermind, all of these samples were taken from lobectomies which means the cancers were removed already.

so from the metadata we actually do not have information about rate or amount of smoking, so it is hard to judge that.
Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
