---
title: "Analysis Report 2: Gender-based Prognostic Analysis of Gene Drivers in Patients with Lung adenocarcinoma"
author: "Alfredo Hans Locht"
date: "November 10, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

## Background

Lung cancer has been identified worldwide as the most prevalent type of cancer with the highest mortality rates in developed countries [@Sakurai2010; @li2015rna; @seo2012transcriptional]. Tobacco cigarette smoking has been identified as the main contributer, along with a host of other genetic and environmental factors, to cause Lung adenocarcinoma, the most common type of non-small cell lung cancer (NSCLC). Genetic research in this field has mainly focused on finding the gene-expression differences between non-smokers and smokers with NSCLC and even between smokers who have and have not contracted any form of NSCLC [@Cheng2012; @li2015rna; @seo2012transcriptional]. These studies have found significant gene expression profile differences between smokers and nonsmokers with NSCLC; however, these studies have not addressed gender-based expression differences.

Although there has been an over 500% increase in lung cancer rates among women since the 1980's, it is becoming increasingly evident that women contain gender-based genetic differences that predicate a favorable prognosis and higher survival rate than men when normalizing for other factors [@Chatkin2004; @Sakurai2010]. These gender-based differences have only been observed qualitatively 

## Objective



# Methods

## Sample origin and sequencing

The dataset generated by Seo et al. for their journal article, The transcriptional landscape and mutational profile of lung adenocarcinoma, was obtained by collecting 200 specimens from 87 patients inflicted with lung adenocarcinoma who had undergone major lung resection surgeries at Saint Mary's Hospital in Seoul, South Korea. The patients provided their smoking history which included smokers (47), non-smokers (36), and some who did not not disclose their smoking history (4). All of the samples originated from people of Korean ethnicity, and out of the 200 specimen, 87 were identified as cancerous stemming from driver mutations while 77 samples were identified to be normal tissue as a basis for comparison of gene expression profiles against the cancerous samples. Together, these 164 were RNA-sequenced using an Illumina HiSeq 2000 sequencer yielding 14,038,673,860 paired-end, 101-bp long reads [@seo2012transcriptional]. Additionally, genomic DNA was extracted from normal lung tissue for exome sequencing. Together, the RNA and exome sequencing reads were aligned to the NCBI human reference genome (build 37.1) using GSNAP with a 5% allowance for mismatches. Image analysis and basecalling was performed using the Illumina pipeline on the default settings (v1.8).

Li et al. obtained 68 transcriptome sequencing data from the aforementioned Seo et al. study

Add about half a page here. In this section instead of first person (I/we), use Li et al. [@li2015rna] and Seo et al. [@seo2012transcriptional], since you'll just be describing what they did, based on the methods in their paper(s). It's important to include this so the reader knows what the study design was for the data you will be presenting.

## Computational

These are the methods that were used to process the sequencing data. Should probably be at least a half of a page. At a very minimum should include citations for biomartr, trimmomatic, and sailfish. Note that these three methods references don't count towards the five references you need to cite in the introduction.

# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```

```{r}
EGFR <- final_table %>%
  filter(genename == "EGFR") %>%
  group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

EGFR %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))

```

```{r}
KRAS <- final_table %>%
  filter(genename == "KRAS") %>%
  group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

KRAS %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))

```

```{r}
MET <- final_table %>%
  filter(genename == "MET") %>%
group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = 
              mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) 

MET %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge",
             aes(color = age_at_diagnosis)) 
  
```

```{r}
IGF1R <- final_table %>%
  filter(genename == "IGF1R") %>%
  group_by(gender, 
           age_at_diagnosis, 
           cancer_stage) %>%
  summarize(mean_count = 
              mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

IGF1R %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = age_at_diagnosis))
```



```{r}
HER2 <- final_table %>%
  filter(genename == "HER2") %>%
  group_by(gender, age_at_diagnosis, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count))

HER2 %>%
  ggplot(aes(x = cancer_stage,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge", aes(color = gender))
```


**Figure 1**: Here we show an example figure caption.



**Figure 2**: Here we show another example figure caption.

# Discussion

Limitations: We have the patient data, but do not know if they have had any kind of previous operation to extract or get rid of the cancer in the metadata or what kind of treatment (if any) they have undergone. --> nevermind, all of these samples were taken from lobectomies which means the cancers were removed already.

so from the metadata we actually do not have information about rate or amount of smoking, so it is hard to judge that.
Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
